backend/config.js

```diff
@@ -34,7 +34,9 @@ const config = {
       useNewUrlParser: true,
       useUnifiedTopology: true,
     },
-    uri: `mongodb://${configData.MONGO_URI}`,
```

Comment:
I finally got this working; we need to use `mongodb+srv://` as the protocol for MongoDB Atlas: https://docs.mongodb.com/manual/reference/connection-string/

I modified this to conditionally prepend the protocol if it does not exist, otherwise it does not prepend the protocol.